commerce-lang/src/main/resources/content/Language.properties

```diff
@@ -224,11 +224,13 @@ gtin-help=Enter the Global Trade Item Number (GTIN). These identifiers include U
 gtin-x=GTIN: {0}
 image-extensions=Allowed Image Extensions
 image-max-size=Image Maximum Size
+import-regions=Import Regions
 inventory=Inventory
 inventory-engine=Inventory Engine
 inventory-engine-method=Inventory Engine Method
 invite-user=Invite user
 invite-user-to-this-account=Invite User to This Account
+italy=Italy
```

Comment:
we should use the OOTB portal keys for this -- `country.italy`, `country.united-states`, etc.